lib/hammer_cli_katello/repository.rb

```diff
@@ -70,6 +70,10 @@ class InfoCommand < HammerCLIKatello::InfoCommand
           field :docker_manifest_total, _("Docker Manifests"), Fields::Field, :hide_blank => true
           field :docker_tag_total, _("Docker Tags"), Fields::Field, :hide_blank => true
         end
+        collection :ostree_branches, _("RPM OS Tree Branches"),
```

Comment:
Also
```
$ hammer repository info --id=25
ID:                   25
Name:                 marquez
Label:                marquez
Organization:         Default Organization
Red Hat Repository:   no
Content Type:         ostree
URL:                  http://sparta.usersys.redhat.com:2025
Publish Via HTTP:     no
Published At:         https://galaxy.example.com/pulp/ostree/web/Default_Organization-ostree-marquez
Relative Path:        Default_Organization/Library/custom/ostree/marquez
Product:              
    ID:   1
    Name: ostree
Sync:                 
    Status: Error
Created:              2016/03/04 14:22:43
Updated:              2016/03/04 14:22:47
Content Counts:       

RPM OS Tree Branches: 
    fedora-atomic/f21/x86_64/updates-testing/docker-host
    fedora-atomic/f21/x86_64/updates/docker-host
```
I see 2 branches there but no value for content unit counts. Might need to fix that to accept ostree_branch